dcmdump/src/main.rs

```diff
@@ -199,7 +251,12 @@ where
                 dump_item(&mut *to, item, width, depth + 2)?;
             }
             to.write_all(&indent)?;
-            writeln!(to, "(FFFE, E0DD) SequenceDelimitationItem",)?;
+            writeln!(
+                to,
+                "{} {}",
+                DumpValue::TagNum("(FFFE, E0DD)"),
```

Comment:
Tags are displayed without spaces, one might as well fix existing inconsistencies.
```suggestion
                DumpValue::TagNum("(FFFE,E0DD)"),
```